datanode/gateway/graphql/schema.graphql

```diff
@@ -4568,3 +4571,40 @@ input DateRange {
 type ProtocolUpgradeStatus {
   ready: Boolean!
 }
+
+enum ProtocolUpgradeProposalStatus {
```

Comment:
Can you put a description on this enum category?

Reply:
Yep sure